### PR TITLE
advance harbor to show also CWE #8632

### PR DESCRIPTION
**Pull request by manuel-sommer**
Advance Harbor parser to show CWE, see linked issue

### CODE REVIEW

**Comment by manuel-sommer** [comment]
What is your opinion, do we need to add also the Scanner information (Trivy vs. Clair)? If yes, where should I add it?

**Comment by manuel-sommer** [comment]
Please review @damiencarol and @mtesauro. Easy review.

File: unittests/tools/test_harbor_vulnerability_parser.py [hunk 1]
```diff
@@ -44,3 +44,13 @@ def test_parse_file_with_multiple_vuln_has_multiple_findings(self):
...
+
+    # Sample with Trivy Test
+    def test_parse_file_with_multiple_vuln_has_multiple_trivy_findings(self):
+        testfile = open("unittests/scans/harbor_vulnerability/harbor-trivy-vuln.json")
+        parser = HarborVulnerabilityParser()
+        findings = parser.get_findings(testfile, Test())
+
+        finding = findings[0]
+        self.assertEqual(finding.severity, 'High')
+        self.assertEqual(finding.cwe, '125')
```
Comment (manuel-sommer) [inline unittests/tools/test_harbor_vulnerability_parser.py:56]: Why do I need a string here for finding.cwe and in the veracode an integer:
https://github.com/DefectDojo/django-DefectDojo/blob/9742d9ff21b36ef7daf6b927ed5ac822c8227c49/unittests/tools/test_veracode_parser.py#L30C18-L30C18 

`self.assertEqual(123, finding.cwe)`

Comment (manuel-sommer) [reply unittests/tools/test_harbor_vulnerability_parser.py:56]: Also here there is an example equally to veracode:
https://defectdojo.github.io/django-DefectDojo/contributing/how-to-write-a-parser/#tests-api-parsers

**Comment by manuel-sommer** [comment]
@Maffooch @cneill @mtesauro 
Could you review his PR please?

**Review by mtesauro — APPROVED** [review]
Approved

**Comment by manuel-sommer** [comment]
@cneill and @devGregA please review :-)

**Review by cneill — APPROVED** [review]

**Comment by manuel-sommer** [comment]
One review more to go @devGregA or @damiencarol

**Review by blakeaowens — APPROVED** [review]

**Pull request merged**